verify_test_type/README.md

```diff
@@ -0,0 +1,43 @@
+<h1 align="center">Verify Test Type</h1>
+
+<p align="center">Script Purpose: Ensure the identification of the test type (e.g., ID, OD, NOD, etc.) for new tests discovered in a repository. </p>
+
+## Steps to Run the Script locally
+
+#### 1. Go to the root directory
+
+Go to the root directory of the repository which has the newly found flaky test.
+
+#### 2. Download the script in your repository
+
+Ensure you have wget installed on your system. You can install it using the package manager of your operating system:
+- For Ubuntu/Debian: sudo apt-get install wget
+- For macOS (using Homebrew): brew install wget
+- For Windows: Download it from GNU Wget.
+
+Now download the script using this command:
+
+```
+wget https://raw.githubusercontent.com/TestingResearchIllinois/idoft/main/verify_test_type/verify_test_type.sh
+```
+
+#### 3. Make the script executable
+
+Use this command to make the script executable:
+
+```
+chmod +x verify_test_type.sh
+```
+
+#### 4. Run the script
+
+Run the script for a particular test using this command:
+
+```
+./verify_test_type.sh <module-name> <commit-hashcode> <test-name> <class-name>
+```
+
+Example of an invocation:
+```
+./verify_test_type.sh bundles/org.openhab.core.thing 660102e3f93f928473b66f56f2955090dfa3c30e testCreateChannelDescriptionChangedEventOnlyNewValue ThingEventFactoryTest
```

Comment:
Since you're provided repo and commit SHA here, what ensures that the exact commit is checked? Does the script clone the repo? Or does it assume that you've already cloned it? If the latter, does it check that the commit SHA matches `git rev-parse HEAD`?

Reply:
The repository is forcefully reset to the exact commit SHA specified by $commit_hash. This guarantees that all operations performed by the script will use this specific state of the repository. It assumes the repository is already cloned.

Reply:
Good description. Please add it in `README` (on in the script), because the users won't be reading this PR discussion.